srcpkgs/reddio/INSTALL.msg

```diff
@@ -0,0 +1 @@
+Optional: netcat to authenticate with your Reddit account credentials
```

Comment:
we don't really use INSTALL.msg for optional dependencies and this seems to be an important dependency. I would remove INSTALL.msg and include it in $depends.